modules/administration-guide/partials/proc_enabling-and-exposing-che-metrics.adoc

```diff
@@ -1,21 +1,26 @@
 // monitoring-{prod-id-short}
 
 [id="enabling-and-exposing-{prod-id-short}-metrics_{context}"]
-= Enabling and exposing {prod-short} metrics
+= Enabling and exposing {prod-short} server JVM metrics
 
-This section describes how to enable and expose {prod-short} metrics.
-
-.Procedure
-
-. Set the `CHE_METRICS_ENABLED=true` environment variable, which will expose the `8087` port as a service on the `che-master` host.
-
-When {prod} is installed from the OperatorHub, the environment variable is set automatically if the default `CheCluster` CR is used: 
-
-image::monitoring/monitoring-che-che-cluster-cr.png[link="{imagesdir}/monitoring/monitoring-che-che-cluster-cr.png"]
+When {prod} is installed from the OperatorHub using the default `CheCluster` CR, the `metrics` environment variable is set automatically to `enable: true` in the `CheCluster` CR: 
 
 [source,yaml]
 ----
 spec:
   metrics:
     enable: true
 ----
+
+.The `metrics` environment variable enabled by default in the `CheCluster` CR
+====
+image::monitoring/monitoring-che-che-cluster-cr.png[link="../_images/monitoring/monitoring-che-che-cluster-cr.png"[]
+====
+
+To manually enable and expose the {prod-short} JVM metrics:
+
+.Procedure
+
+* Set the `CHE_METRICS_ENABLED=true` environment variable.
```

Comment:
Question: where/how/UI?/tool? (in a few words) is this environment variable set?

Reply:
Good point, this section should be updated again (which I've done in my latest commit). The environment variable is set by changing the CheCluster CR.

Here is a preview of the docs:
<img width="1135" alt="image" src="https://user-images.githubusercontent.com/83611742/164525868-4a9e4880-9f1f-405e-b7b7-7a4c6fbb9f32.png">

Reply:
I've taken it a bit further.
Feel free to resolve this thread.